src/fosslight_prechecker/_constant.py

```diff
@@ -7,7 +7,7 @@
                              "pfm", "otf", "afm", "dfont", "json"]
 OSS_PKG_INFO_FILES = [r"oss-pkg-info[\s\S]*.yaml", "oss-pkg-info.yml", r"oss-package[\s\S]*.info", "requirement.txt",
                       "requirements.txt", "package.json", "pom.xml", "build.gradle", "podfile.lock", "cartfile.resolved",
-                      "pubspec.yaml", "package.resolved", "go.mod", r"fosslight-sbom-info[\s\S]*.yaml"]
+                      "pubspec.yaml", "package.resolved", "go.mod", r"fosslight-sbom-info[\s\S]*.yaml", r"sbom-info[\s\S]*.yaml"]
```

Comment:
How about removing the followings?
- r"fosslight-sbom-info[\s\S]*.yaml
-  "oss-pkg-info.yml"